smap/sources/safarScraper.py

```diff
@@ -58,3 +59,30 @@ def get_air_quality_data(opts):
 		logging.debug("Traceback : "+ str(traceback.format_exc()))
 		air_quality_data = []
 		return air_quality_data
+
+def get_air_quality_init(opts):
+	"""
+	Returns only the required info about the safar sensors
+	"""
+	try:
+		url = 'http://safar.tropmet.res.in/map_data.php?for=current&city_id=2'
+		content = get_content(url)
+		x = ast.literal_eval(content)
```

Comment:
Can't you use the json library instead of this? `ast.literal_eval` seems like overkill.